frontend/src/app/containers/ExperimentPage.js

```diff
@@ -276,10 +276,11 @@ export class ExperimentDetail extends React.Component {
                 <h2 className="banner">
                   <span data-l10n-id="experimentPromoHeader" className="block">Ready for Takeoff?</span>
                 </h2>
-                <p data-l10n-id="experimentPromoSubheader">We're building next-generation features for Firefox. Install Test Pilot to try them!</p>
+                <p data-l10n-id="experimentPromoSubheader">We&apos;re building next-generation features for Firefox. Install Test Pilot to try them!</p>
```

Comment:
The dangling apostrophe was just screwing up diffs on github. I think a better change is just to remove the text instead.